### DISCUSSION TITLE
A question about python interfaces

### DISCUSSION THREAD

**Discussion by jiek10** [lbl-srg/modelica-buildings#3844] [Q&A] [answered]
I am trying to use the Python_3_8 interface, I want to import pandas which is an external package installed by pip to read data.I find that when I use  external package in python, it will run incorrectly in Dymola. But there is nothing wrong with using only python's native functions , such as import random. I don't know if there's something wrong with my code or if my path is set up incorrectly. Any suggest?
Thank you
![3](https://github.com/lbl-srg/modelica-buildings/assets/170920836/f5af32c6-f31a-4110-b6ac-8e6d21ad8dd6)
![1](https://github.com/lbl-srg/modelica-buildings/assets/170920836/ca143ae3-856e-49d9-82fd-eba3af23409d)
![1716799858641](https://github.com/lbl-srg/modelica-buildings/assets/170920836/a9510737-029f-4fe6-b7a0-05df6788fd8f)

**Comment by mwetter** [comment]
I think using pandas should work. Perhaps you have a typo as your path contains `pythorn_csv` which maybe should be `python_csv`? Please confirm if this fixes it.

**Reply by jiek10** [reply, marked as answer]
Thank you very much for your reply. I then tried to call pandas and sklearn in C and found that there was a lack of dynamic library files, then I tried to put the DLLs file in the python directory into the system path, and finally the python interface worked.
